rocky/reports/runner/worker.py

```diff
@@ -224,6 +225,8 @@ def _start_working(
             except HTTPError:
                 logger.exception("Could not patch scheduler task to %s", status.value)
 
+            gc.collect()
```

Comment:
Is it me, or do we not have any sleep() in this loop?

Reply:
`task_queue.get()` is blocking by default.